### PR TITLE
feat(ci): connect v9 production release setup

### PR DESCRIPTION
**Pull request by vdovhanych**
Sets up connect v9 release to production from suite monorepo. Using its own deploy runner and configuration. 
Release to production will be possible only from the branch `release/connect-v9`

branch `release/connect-v9` should be protected and only select people should be able to push and merge content there. 

I would also keep the exact same setup as we had in connect. Meaning PR from develop to release/connect-v9 only and not from any other branches.

Next steps

- [ ] Branch `release/connect-v9` but after this is merged
- [ ] Setup that branch as protected
- [ ] Update docs?

### CODE REVIEW

File: ci/releases.yml [hunk 1]
```diff
@@ -99,3 +99,40 @@ msg-system codesign deploy:
...
     - aws cloudfront create-invalidation --distribution-id ${AWS_CLOUDFRONT_ID} --paths '/config/stable/*'
+
+# connect v9 deploy to production jobs
+
+# Create rollback copy of connect.trezo.io
+connect v9 rollback production:
+  stage: deploy to production
+  only:
+    refs:
+      - release/connec-v9
```
Comment (mroz22) [inline ci/releases.yml]: maybe typo? connect

File: ci/scripts/connect-release.sh [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+# @DESTINATION: required, destination directory (connect major version)
+
+# Validate params
+if [ -z "$1" ]; then
+    echo "Invalid params"
+    echo "Usage example: ./s3sync.sh 9"
+    exit 1
+fi
+
+echo "Uploading to s3://connect.trezor.io/$1/"
```
Comment (mroz22) [inline ci/scripts/connect-release.sh]: personally, I would asign some default (=9) to $1 so that we dont screw up accidentaly

Comment (mroz22) [reply ci/scripts/connect-release.sh]: maybe even forbid anything different than `9` at the moment. I am just super anxious when touching these production-grade things

File: ci/releases.yml (hunk 1 above)
Comment (vdovhanych) [reply ci/releases.yml]: yep, will fix it

**Review by mroz22 — APPROVED** [review]
just few nitpicks / ideas

**Review by mroz22 — APPROVED** [review]
just few nitpicks / ideas

File: ci/releases.yml (hunk 1 above)
Comment (vdovhanych) [reply ci/releases.yml]: fixed

File: ci/scripts/connect-release.sh (hunk 2 above)
Comment (vdovhanych) [reply ci/scripts/connect-release.sh]: The script will now accept only 9 as a parameter. Please test it at your end to make sure its working as expected.

**Comment by mroz22** [comment]
nice, lets try and see

**Comment by vdovhanych** [comment]
Some updates to the deploy. 
Release job will now create two releases one in the `/9/` folder (that's always latest) and to the current version folder f.e. `/9.0.1/` It's something we were discussing doing for better ability to revert it if something brakes in the latest release. 
I will now test it on the testing s3 bucket and we will see what it does there.

**Review by mroz22 — APPROVED** [review]

**Pull request merged**